src/pages/index.js

```diff
@@ -14,4 +14,8 @@ const IndexPage = () => {
   )
 }
 
+export const Head = ({ data }) => (
```

Comment:
This page has no data prop. 

We may need to introduce a small SEO component that uses static so we'd only need to pass the title. 

Seem like we link this repo to a tutorial and just changing stuff that way may not make sense?? So far I can't see anywhere where we link this repo though

Reply:
We're changing this note in https://github.com/gatsbyjs/gatsby/pull/35893/files#diff-c339dc52818c45ab2d08752adf550916cb56c857762499ed38e524ac269f7fa7 already, so this should be done the same if applicable in the main PR